src/main/java/org/fibsters/CoordinatorComputeEngineImpl.java

```diff
@@ -11,7 +11,7 @@
 public class CoordinatorComputeEngineImpl implements CoordinatorComputeEngine {
 
     private FibCalcComputeEngineImpl fibCalcComputeEngine;
-    private final FibSpiralComputeEngineImpl fibSpiralComputeEngine;
+    private FibSpiralComputeEngineImpl fibSpiralComputeEngine;
```

Comment:
We probably dont need a reference to any sort of fibspiral engine or the fibcalcengine here. since it should make these dynamically.